lib/modules/manager/cargo/extract.ts

```diff
@@ -184,11 +189,27 @@ export async function extractPackageFile(
       targetDeps = targetDeps.concat(deps);
     });
   }
+
+  const workspaceSection = cargoManifest.workspace;
```

Comment:
why this needs to be so complicated? can't we use same schema as the other normal dependencies section?

Reply:
This works pretty much the same way as target deps above. As such it is more of a new group of deps rather than type such as dev, build deps

Reply:
I've removed some of the redundant sections left over from where I initially got it from. Tell me what you think =D